cluster-autoscaler/simulator/cluster.go

```diff
@@ -62,7 +64,8 @@ type NodeToBeRemoved struct {
 type UtilizationInfo struct {
 	CpuUtil float64
 	MemUtil float64
-	// Max(CpuUtil, MemUtil).
+	GPUUtil float64
```

Comment:
to follow the `CpuUtil` convention this should probably be `GpuUtil`

Reply:
Resolved.